Makefile

```diff
@@ -0,0 +1,43 @@
+.PHONY: default release release-test release-upload
+
+default:
+	@echo "This is only for building releases thus far"
+	@echo "Select:"
+	@echo "   make release"
+	@echo "   make release-test"
+	@echo "   make release-upload"
+	@echo "It will run 'pip install' in your current venv"
+
+release:
+	@echo "Is checkout clean?"
+	git diff-files --quiet
+	git diff-index --quiet --cached HEAD --
+
+	@echo "Install required build software"
+	pip install --editable .[build]
```

Comment:
Again, consider doing all of this in a virtualenv (and using `python -m pip`).

Reply:
it does say up top that it'll do it "in the current venv" .. but maybe it's better to do it in a "clean" one anyway...